.travis.yml

```diff
@@ -38,5 +36,6 @@ script:
       fi
 
     - if [[ $TEST_TARGET == 'coding_standards' ]]; then
-        find . -type f -name "*.py" ! -name 'conf.py' | xargs flake8 --max-line-length=100 ;
+        pycodestyle --ignore=E741,E226 --max-line-length=100 tests ;
```

Comment:
We could use a `tox.ini` (or a `setup.cfg` I believe) file instead to specify all the `pycodestyle` options. We could add `pytest` option to there as well, if we should ever need any

Reply:
I don't really like `tox.ini` and `setup.cfg` for that purpose b/c make it harder to understand what is going on. I like to have all the test related configuration in a single place, the `.travis.yml` file.

With that said I don't have a strong opinion on that, it is just my preference. If you want me to make the change I can do that.

Reply:
Ah, I also didn't see which file this was in. At my day job, if we threw everything into the travis.yml, it would become unruly so quickly, which is where the suggestion comes from. I am fine with it here though :+1:

Reply:
Indeed there are some case where the Travis-CI config file is so complex that a "re-factor" is needed (build/test scripts, tox/setup.cfg, etc), but I don't think it is the case here. Our's is very readable :smile: